nfcore_custom.config

```diff
@@ -20,6 +20,7 @@ profiles {
     bigpurple      { includeConfig "${params.custom_config_base}/conf/bigpurple.config" }
     binac          { includeConfig "${params.custom_config_base}/conf/binac.config" }
     biohpc_gen     { includeConfig "${params.custom_config_base}/conf/biohpc_gen.config" }
+    BUTTER_HPC     { includeConfig "${params.custom_config_base}/conf/BUTTER_HPC.config" }
```

Comment:
Can the profile be lowercased, and so for the file name?

Reply:
Yes I will take care of that and let you know when finished.

Reply:
Done lowercasing all relevant file names and contents!

Reply:
still hasn't changed

Reply:
I took care of this now